src/components/MessagesList/MessagesGroup/Message/MessagePart/DeckCard.vue

```diff
@@ -73,6 +83,54 @@ export default {
 				board: this.boardname,
 			})
 		},
+		deckCardAriaLabel() {
+			return t('spreed', 'Deck Card')
+		},
 	},
 }
 </script>
+
+<style lang="scss" scoped>
+.deck-card {
+	display: flex;
+	transition: box-shadow 0.1s ease-in-out;
+	box-shadow: 0 0 2px 0 var(--color-box-shadow);
+	border-radius: var(--border-radius-large);
+	font-size: 100%;
+	background-color: var(--color-main-background);
```

Comment:
![Bildschirmfoto von 2021-02-03 16-19-51](https://user-images.githubusercontent.com/213943/106767942-b58dba00-663b-11eb-9efb-c351072075ae.png)

Might be my personal preference, but the "cut out" look makes it feel a bit weird when hovering on dark mode?

Reply:
On on non-hover it looks just like it would be out of place position, as the boarder is hardly visible

Reply:
addressed!